_posts/2017-05-30-database-encryption.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Database encryption"
+date: 2017-05-30 11:03
+comments: true
+author: ashkan
+categories: [Database, Encryption, Security]
+---
+
+Recently after examining the data stored in one of our systems, we noticed that while originally this system wasn't designed to include sensitive data, over the time it ended up including some sensitive information and we need to encrypt the data.
```

Comment:
over time (remove the)

Maybe replace "sensitive information" by "potentially sensitive user data"?